Mathlib/Order/Monotone/Monovary.lean

```diff
@@ -278,9 +278,6 @@ end PartialOrder
 
 variable [LinearOrder ι]
 
-/- Porting note: Due to a bug in `alias`, many of the below lemmas have dot notation removed in the
-proof -/
```

Comment:
Not sure at all what this was even supposed to refer to: in Mathlib3 these were all `lemma`s too...